packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -47,6 +49,34 @@ class DescopeWc extends BaseDescopeWc {
 
   flowState: State<FlowState>;
 
+  log = {
+    error: (message: string, description?: string) => {
+      super.logger().error(message, description);
+      this.#dispatchLog({
```

Comment:
Do we think the descopers will use this event or it's only for internal use?

Reply:
yes, this is currently the only option for descopers to know what happening in the sdk, this should be useful.